README.md

```diff
@@ -507,6 +509,526 @@ source scripts/makeAllPlots.sh figures/ResponseUniformityMaps
+	Canv_Margin_Bot = '0.14';
+	Canv_Margin_Lf = '0.16';
+	Canv_Margin_Rt = '0.06';
+	Canv_Name = 'LS2_Detectors';
+	Canv_Plot_Type = 'TGraphErrors';
+	Canv_Range_X = '0,1000';
+	Canv_Range_Y = '0,7';
+	Canv_Title_Offset_X = '1.0';
+	Canv_Title_Offset_Y = '1.0';
+	Canv_Title_X = '';
+	Canv_Title_Y = '';
+		[BEGIN_PLOT]
+			Plot_Color = 'kRed';
+			Plot_LegEntry = 'GE11-X-L-CERN-0002';
+			Plot_Line_Size = '1';
+			Plot_Line_Style = '1';
+			Plot_Marker_Size = '0.8';
+			Plot_Marker_Style = '20';
+			Plot_Name = 'GE11-X-L-CERN-0002';
+			[BEGIN_DATA]
+			VAR_INDEP,VAR_DEP,VAR_INDEP_ERR,VAR_DEP_ERR
+			1.0,20.38,0.0,0.0
+			62.0,20.26,0.0,0.0
+			123.0,20.13,0.0,0.0
+			184.0,20.14,0.0,0.0
+			245.0,20.11,0.0,0.0
+			306.0,20.11,0.0,0.0
+			367.0,20.04,0.0,0.0
+			428.0,20.05,0.0,0.0
+			489.0,20.03,0.0,0.0
+			550.0,20.07,0.0,0.0
+			...................
+			...................
+			...................
+			2990.0,19.39,0.0,0.0
+			3051.0,19.33,0.0,0.0
+			3112.0,19.39,0.0,0.0
+			3173.0,19.32,0.0,0.0
+			3234.0,19.32,0.0,0.0
+			3295.0,19.35,0.0,0.0
+			3356.0,19.23,0.0,0.0
+			3417.0,19.25,0.0,0.0
+			3478.0,19.25,0.0,0.0
+			3539.0,19.29,0.0,0.0
+			3600.0,19.2,0.0,0.0
+			[END_DATA]
+			[BEGIN_FIT]
+			Fit_Color = 'kRed';
+			Fit_Formula = '[0]';
+			Fit_LegEntry = 'Fit_GE11-X-L-CERN-0002';
+			Fit_Line_Size = '1';
+			Fit_Line_Style = '1';
+			Fit_Name = 'Fit_GE11-X-L-CERN-0002';
+			Fit_Option = 'R';
+			Fit_Param_IGuess = '0';
+			Fit_Perform = 'true';
+			Fit_Range = '0,1000';
+			[END_FIT]
+		[END_PLOT]
+	[END_CANVAS]
+	
+3) To produce the graph execute the command:
+
+
+	 ./genericPlotter Myconfig.cfg true
+
+	 
+### 3.b.ii.II Helper Script - Parsing Excel Files to genericPlotter - Additional Scripts
+
+**Additional scripts:** The additional scripts are designed to give specific instructions to the main script about the style and the columns plotted depending on the kind of test. For example if the QC3.py script is selected the columnY= 2 (corresponds to the pressure(mbar) column) vs columnX= 1 (corresponds to the time(s) column) will be plotted. Moreover the title axis names are set and the latex lines: LS2 Detector Production and Gas=CO2 will be plotted on the graph. The user is able to open and modify the produced config file in case some of the options that are set are not the desirable ones.  
+
+The following options are already set for **QC3.py, QC4\_HV.py, QC4\_SS.py, QC5\_Gain.py, QC5\_Rate.py** scripts:
+
+	
+**QC3.py :**   
+
+	--CanvTitleX=Time #left(s#right)
+	--CanvTitleY=Pressure #left(mbar#right)
+	--SelectColumnX=1
+	--SelectColumnY=2
+	--SelectRowStart=1
+	--SelectRowEnd=61
+	--CanvRangeX=0,3600
+	--CanvRangeY=0,35
+	--LatexLines=0.62,0.86, #splitline{LS2}{Detector~Production}
+	--LatexLines=0.62,0.27, Gas~=~CO_{2}
+
+	Moreover if the Fit option is used the following options are set for the fit:
+
+	--Fit
+	--FitFormula=[0]*TMath::Exp([1]*x)
+	--FitParamIGuess=AMPLITUDE,6.2e-05
+	--FitRange=0,3600
+	
+Example of use: 
```

Comment:
same comment as above...restructure to be an easier copy/paste.

First give the block of code and then describe it.

Reply:
done